Driverless AI/risk-assessment-tools-in-the-world-of-ai-the-social-sciences-and-humanities/risk-assessment-tools-in-the-world-of-ai-the-social-sciences-and-humanities.md

```diff
@@ -0,0 +1,475 @@
+
+5. Click on the SET option 
+
+![sensitivity-analysis-3-african-american](assets/sensitivity-analysis-3-african-american.jpg)
+
+Scroll to the left of the table, and you will be able to see the rest of the table and the new column added. On the left side, you will see the filter section of the sensitivity analysis page, and in there, filter with the following options: race == African-American. 
+
+![sensitivity-analysis-4-residuals](assets/sensitivity-analysis-4-residuals.jpg)
+
+In the sensitivity analysis graph, change the [Target] option to [Residuals]. 
+
+![sensitivity-analysis-5-false-positive](assets/sensitivity-analysis-5-false-positive.jpg)
+
+In the filter section, click on the FP option; this will select the African American group's false positives. After clicking the FP option, the graph should look similar to the image above, where the closer you are to the cut-off line, the closer you were in the model to being predicted as a false two_year_recid. 
+
+![sensitivity-analysis-6-close-to-cut-off.jpg](assets/sensitivity-analysis-6-close-to-cut-off.jpg)
+
+Rather than looking at every instance of false positives in the African American group, let's look at the instances that are super close to the cut-off line and see if the priors_count column is truly driving the model behavior. Reminder: sensitivity analysis enables us to tinker with various data settings to see their weight in a prediction.  
+
+1. To get close to the instances that are close to the cut-off line, I filtered [ID] < 5 (it might be the case that you will have to filter with other options to get close) 
+2. The table now represents the five instances that are close to the cut-off line.
+3. The graph will now reflect the above table's information showcasing the five instances close to the cut-off line.
+
+![sensitivity-analysis-7-id-4](assets/sensitivity-analysis-7-id-4.jpg)
+
+In prediction number 4 ([ID] 4), as highlighted in the image above, we see an African American that was never arrested within the two-year mark, but the model predicted the opposite: a wrong prediction. In this case, we see a male with four prior counts and in the greater than 45 age category. Let's see if having four prior counts was the deciding factor.  
+
+![sensitivity-analysis-8-change-prediction](assets/sensitivity-analysis-8-change-prediction.jpg)
+
+Change the priors_count value from four to zero. Right after, click on the rescore option (on the top left corner where the table is located); something similar to the above page should appear. Here we see that the prediction has been flipped, and as well we come to realize that this person was judged based on his prior counts. It is worth noting that this person perhaps obtained such four prior counts because of racism in our judicial system and, therefore, was predicted by the model as a true two_year_recid. I am not saying that’s true, but given our biased judicial system, that’s not hard to believe. It's something to think for now. 
```

Comment:
Can you please rephrase the last statement to change it from "I" to a more general observation?

Reply:
Thanks